openpype/hosts/tvpaint/plugins/create/create_render_custom.py

```diff
@@ -0,0 +1,171 @@
+            "active": self.active_on_create
+        }
+
+        new_instance = CreatedInstance(
+            self.family, subset_name, data, self
+        )
+        instances_data = self.host.list_instances()
+        instances_data.append(new_instance.data_to_store())
+        self.host.write_instances(instances_data)
+        self._add_instance_to_context(new_instance)
+        return new_instance
+
+    def create(self):
+        existing_instance = None
+        for instance in self.create_context.instances:
+            if instance.creator_identifier == self.identifier:
+                existing_instance = instance
+                break
+
+        if existing_instance is None:
+            return self._create_new_instance()
+
+        create_context = self.create_context
+        host_name = create_context.host_name
+        project_name = create_context.get_current_project_name()
+        asset_name = create_context.get_current_asset_name()
+        task_name = create_context.get_current_task_name()
+
+        if (
+            existing_instance["asset"] != asset_name
+            or existing_instance["task"] != task_name
+        ):
+            asset_doc = get_asset_by_name(project_name, asset_name)
+            subset_name = self.get_subset_name(
+                existing_instance["variant"],
+                task_name,
+                asset_doc,
+                project_name,
+                host_name,
+                existing_instance
+            )
+            existing_instance["asset"] = asset_name
+            existing_instance["task"] = task_name
+            existing_instance["subset"] = subset_name
+
+        existing_instance["label"] = self._get_label(existing_instance["subset"])
+
+    def _get_label(self, subset_name):
+        try:
+            subset_name = subset_name.format(**prepare_template_data({}))
+        except (KeyError, ValueError):
+            pass
+
+        return subset_name
+
+    def get_dynamic_data(self, variant, *args, **kwargs):
+        dynamic_data = super().get_dynamic_data(variant, *args, **kwargs)
+        return dynamic_data
+
+    def get_instance_attr_defs(self):
+        return [
+            BoolDef(
+                "mark_for_review",
+                label="Review Publish",
+                default=self.mark_for_review
+            ),
+            BoolDef(
+                "review_image_seq",
+                label="Video(off) / Img Seq(on) for Review",
+                default=self.review_image_seq
+            ),
+            BoolDef(
+                "keep_frame_index",
+                label="Keep actual frame index in Files/Review",
+                default=self.keep_frame_index
+            ),EnumDef(
+                "export_type",
+                self.exports_types,
+                label="Export Quick Review Through",
+                default=self.export_type
+            ),
+            UISeparatorDef("layer_options"),
+            UISeparatorDef("layer_options_01"),
```

Comment:
Why two UISeparator?

Reply:
Pour une plus belle visibilité dans le publishTool

Reply:
Maybe add a comment :
```python
UISeparatorDef("layer_options"),
UISeparatorDef("layer_options_01"),  # Second separator for better visibility
```

Because without it I would consider this as a typo and remove it if I see that while looking at the file